src/System.Security.Cryptography.X509Certificates/tests/CertificateCreation/CertificateRequestUsageTests.cs

```diff
@@ -296,8 +296,11 @@ public static void SelfSign_ECC_DiminishedPoint_UseCertKeys()
             }
         }
 
-        [Fact]
-        public static void SerialNumber_AlwaysPositive()
+        [Theory]
+        [InlineData("80")]
+        [InlineData("0080")]
+        [InlineData("00000080")]
+        public static void SerialNumber_AlwaysPositive(string desiredSerial)
```

Comment:
If there's not already a test for "00000000" (or whatever your favorite all-zeros representation is), please add one, since that was a point of concern from a code review comment.